src/app/tutorials/angular2-whatsapp-ionic-cli/index.ts

```diff
@@ -15,7 +15,7 @@ export const ANGULAR2_WHATSAPP_IONIC_CLI: TutorialDefinition = {
   },
   versions: {
     // Git revision => version definition object
-    'master': V2_0_0,
+    'master': V3_0_0,
```

Comment:
In this repo: https://github.com/Urigo/Ionic2CLI-Meteor-WhatsApp , the `master` is still the current tutorial, and `master-history` has you new commits... so you should add here:
```ts
'master': V2_0_0,
'master-history': V3_0_0,
```